Automattic/Edit-Flow
# Fix ef_custom_status_list filter on get_custom_statuses()

## Description

This fixes #586, where filtering custom statuses by user role wasn't working.

## Steps to Test

1. Check out PR.
1. Limit user role to subset of custom statuses as described [here](https://editflow.org/extend/limit-custom-statuses-based-on-user-role/)
1. Click the custom status drop-down while editing/creating a post as that user role, and see which custom statuses you have access to
